angular-daemon/apps/common/js/models/Note.ts

```diff
@@ -53,7 +57,7 @@ export default class Note extends Persistable(Supertype) {
 
   public async save() {
     const txn = this.amorphic.begin();
-    this.setDirty(txn);
+    this.persistorSave({ transaction: txn });
```

Comment:
Why are we switching to persistorSave here? This is a daemon app, so we should be using setDirty

Reply:
ah I think I was testing the pathway of the persistorSave functionality when testing the S3 behavior - just a remnant, will change back